### PR TITLE
Add the headers 1 through 6 as supported style types

### PR DESCRIPTION
**Pull request by steurt**
This adds the headers 1 through 6 as supported style types to mui-rte. In some cases this is better suited than only having the single 'title' control.

By default the headers 1 through 6 are not used, the 'title' control is. This is specified in the new `DefaultControls`.

### CODE REVIEW

**Comment by charlesomer** [comment]
Does this not exist yet still? If so I might be able to have a look as it's something I'd like to be able to use 👍

**Comment by steurt** [comment]
@charlesomer the pull request is ready, it requires a review. What do you think about this @niuware?

File: src/components/Toolbar.tsx [hunk 1]
```diff
@@ -72,8 +83,50 @@ type TToolbarProps = {
...
+        label: 'H5',
+        name: "h5",
+        style: 'header-five',
+        icon: <Looks5Icon />,
+        type: "block"
+    },
+    {
+        label: 'H6',
+        name: "h6",
+        style: 'header-siz',
```
Comment (benyap) [inline src/components/Toolbar.tsx:124]: Typo? Should be `header-six`.

Comment (steurt) [reply src/components/Toolbar.tsx]: @benyap typo indeed, thanks, fixed it.

**Comment by charlesomer** [comment]
@niuware are you able to review this please?

**Comment by niuware** [comment]
I feel this can be done by making custom controls rather than be a part of the core code. Also the proposed icons are not very suggestive, as a user I would expect a list with the available title sizes and pick it from there. Maybe that is a better feature to have in the editor.

**Comment by steurt** [comment]
Alright, so no go.

**Pull request closed without merging**